doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_architecture_reset.rst

```diff
@@ -60,16 +64,21 @@ The following is the reset behavior when the LCS of the nRF54H20 SoC is either i
    * - Reset kind in LCS ``ROT`` or ``DEPLOYED``
      - Reset behavior
    * - ``RESET_SYSTEM``
-     - In the Secure Domain, this reset is mapped to the ``SYS_RESET_REQUEST`` bit, as described in the ARM specification for the *Application Interrupt and Register Controller*.
-       This reset triggers a reset of the Secure Domain, which subsequently resets the entire system.
-
-       In the local domains, this reset is available only if the other domains have a valid configuration in the ``CPUCONF`` register and their access port is unprotected.
-       The PC tool uses the CTRL-AP mailbox to send a local domain reset request to the Secure Domain Firmware (SDFW).
+     - Not supported.
    * - ``RESET_HARD``
      - Initiates a reset using the CTRL-AP through the On-Board Debugger (OBD).
        The register used is ``RESET`` (address offset: ``0x00``).
    * - ``RESET_PIN``
      - J-Link toggles the ``RESET`` pin via OBD.
+   * - ``RESET_VIA_SECDOM``
+     - In the Secure Domain, this reset is mapped to the ``SYS_RESET_REQUEST`` bit, as described in the ARM specification for the *Application Interrupt and Register Controller*.
+       This reset sends a reset request over ADAC to the Secure Domain Firwmare (SDFW).
+       This triggers a reset of the Secure Domain, which subsequently resets the entire system.
+
+       In the local domains, this reset is available only if the other domains have a valid configuration in the ``CPUCONF`` register and their access port is unprotected.
+       The PC tool uses the CTRL-AP mailbox to send a local domain reset request to the Secure Domain Firmware (SDFW).
```

Comment:
I wasn't precise enough in https://github.com/nrfconnect/sdk-nrf/pull/17411#discussion_r1802937633, sorry about that. There is still some incorrect info here, but I think we can just reduce it down to this:

```suggestion
       - Uses the CTRL-AP mailbox to send a local domain reset request to the Secure Domain Firmware (SDFW).
```

If a more detailed description about what SDFW actually does is needed, https://github.com/nrfconnect/sdk-secdom/blob/main/doc/ctrlap_adac.rst#reset---sdfw_adac_reset has a description of it. Copypasting from that, something like 

"  - Uses the CTRL-AP mailbox to send a local domain reset request to the Secure Domain Firmware (SDFW).

SDFW performs a reset of nonessential local domains and global peripherals owned by those domains. Nonessential domains include all local domains except for the Secure Domain. Only domains with a populated UICR are reset. The reset CPUs do not restart after the command is completed."

might work here.

Reply:
Solved, Please let me know if now the description is correct @bjorgan